frontend/src/bundles/video_editor/components/timeline/subcomponents/item.tsx

```diff
@@ -0,0 +1,41 @@
+import { type Span } from 'dnd-timeline';
+import { useItem } from 'dnd-timeline';
+
+interface ItemProperties {
+	id: string;
+	span: Span;
+	children: React.ReactNode;
+}
+const Item = (properties: ItemProperties): JSX.Element => {
```

Comment:
It's better to use destructuring approach

Reply:
Great recommendation. Thank you and solved